README.md

```diff
@@ -104,13 +103,7 @@ where we load a Jitsi video call iframe into Convene's UI.
 If you want to run Jitsi locally, see the configuration provided in
 [`infrastructure/jitsi-meet-local`](./infrastructure/jitsi-meet-local/README.md).
 
-### Architecture overview diagram
-
-_Last updated: Dec 2022_
-
-![](./docs/convene-architecture-overview.jpg)
-
-Original on Miro: https://miro.com/app/board/o9J_lLrbz1g=/
+See [Architecture](./CONTRIBUTING.md#11-architecture) for how Convene is architected.
```

Comment:
I know this sentence is terrible. I could not think of something better right now. Totally open to suggestions.

`See Architecture for how Convene is <insert here>.`

Reply:
I think this is fine for now.